<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainControlsWidget</class>
 <widget class="QWidget" name="MainControlsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="qtabw_tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="qtab_evaluation">
      <attribute name="title">
       <string>Evaluation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4"/>
     </widget>
     <widget class="QWidget" name="qtab_hypfit">
      <attribute name="title">
       <string>Hyperbola fit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
